src/cloudWatchLogs/document/logStreamDocumentProvider.ts

```diff
@@ -54,3 +54,17 @@ export async function highlightDocument(registry: LogStreamRegistry, uri: vscode
         textEditor.setDecorations(HIGHLIGHTER, ranges)
     }
 }
+
+export function getActiveUri(registry: LogStreamRegistry) {
```

Comment:
I think a more verbose name works here because it seems like a pretty important behavior:
```suggestion
export function getActiveDocumentUri(registry: LogStreamRegistry) {
```